BUILD.gn

```diff
@@ -79,6 +79,7 @@ group("flutter") {
       "//flutter/shell/platform/embedder:embedder_unittests",
       "//flutter/shell/platform/glfw/client_wrapper:client_wrapper_glfw_unittests",
       "//flutter/testing:testing_unittests",
+      "//flutter/third_party/skia:skia_unittests",
```

Comment:
This library name can be confused with `//third_party/skia` and is actually just a utility that happen to contain code from Skia. Just call this `sktree` or something indicative of what this library does or is in it.